io.catenax.week_based_material_demand/RELEASE_NOTES.md

```diff
@@ -2,6 +2,12 @@
 
 All notable changes to this model will be documented in this file.
 
+## [3.0.1] - 2024-09-16
```

Comment:
should be the date when you got the MS3 approval.

Reply:
Sorry, that was my bad as I did not add the label for the approval. I will notify the team.

Reply:
Changed